types/index.d.ts

```diff
@@ -249,7 +249,7 @@ type TransformerObject = {
   cache?: boolean | TransformerCacheObject | undefined;
 };
 type Transform = TransformerFunction | TransformerObject;
-type Filter = (filepath: string) => any;
+type Filter = (filepath: string) => boolean;
```

Comment:
We should add this here https://github.com/webpack-contrib/copy-webpack-plugin/blob/master/src/index.js#L95 and run `npm run build`, same for https://github.com/webpack-contrib/copy-webpack-plugin/blob/master/src/index.js#L99

Reply:
Done